plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1298,7 +1298,6 @@ public void testBrokerQueryWithTooManyRowsForSegmentQuery()
 
     @Test
     public void testMaxLimitForPassthroughQueries()
-            throws InterruptedException
```

Comment:
why doesn't it fail on master?

Reply:
Because the exception message no longer matched the expected message since the sql was uppercased.

Reply:
If you mean the exception - there no interrupted exception thrown so no need to check for it, nothing failed just code cleanup. Should I squash it?